Spec/RealtimeClientConnection.swift

```diff
@@ -2390,13 +2395,14 @@ class RealtimeClientConnection: QuickSpec {
                                 done()
                             }
                         }
-                        expect(channel.state).to(equal(ARTRealtimeChannelState.detached))
+                        expect(client.msgSerial).to(equal(0))
+                        expect(channel.state).to(equal(ARTRealtimeChannelState.attaching))
```

Comment:
:+1:
(very minor comment: maybe use `oldConnectionId` or something rather than `expectedConnectionId`, `expect(...).toNot(equal(expectedConnectionId))` reads a bit weirdly :) )